src/components/ClaimFilter.js

```diff
@@ -15,7 +15,7 @@ import {
   AmountInput,
   Contributions,
 } from "@openimis/fe-core";
-import { selectClaimAdmin, selectHealthFacility, selectDistrict, selectRegion } from "../actions";
+import { selectClaimAdmin, selectHealthFacility, selectDistrict, selectRegion, clearClaim } from "../actions";
```

Comment:
This import seems to be too long. Could you reorganize them as you did with actions in the _ClaimMasterPanel.js_? Moreover, what's the purpose of importing here _clearClaim_ action? If I'm not mistaken, you are using it only in _ClaimMasterPanel.js_.